### DISCUSSION TITLE
Supply Chain Factors v1.2 missing 331314?

### DISCUSSION THREAD

**Discussion by gsdipietro** [USEPA/supply-chain-factors#46] [Q&A] [answered]
There are no emission factors for Secondary Smelting and Alloying of Aluminum (331314). Is this correct?

**Comment by bl-young** [comment, marked as answer]
Yes it is correct. The current release is based on a [detail level commodity model](https://github.com/USEPA/supply-chain-factors/blob/main/model-specs/USEEIOv2.1.19-GHG.yml#L10). In these models, 331314 is only an industry and not a commodity (that is, both 331313 and 331314 produce the same commodity: 331313). As a result, in a commodity model, no factors are produced for 331314.

**Comment by gsdipietro** [comment]
Great, thank you.
